### PR TITLE
feat: Add capabilities in batch client get enrichment activities to use Enrichment flags

### PR DESCRIPTION
**Pull request by adinovri**

### CODE REVIEW

**Review by xernobyl — APPROVED** [review]
LGTM

**Comment by adinovri** [comment]
Hi @xernobyl @JcMinarro when will this changes can be release in new version? Thanks.

**Comment by xernobyl** [comment]
> Hi @xernobyl @JcMinarro when will this changes can be release in new version? Thanks.

I'll try to have it done in the next 8 hours.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -16,8 +16,8 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Commit message lint
-        uses: wagoid/commitlint-github-action@v4
+      #- name: Commit message lint
+      #  uses: wagoid/commitlint-github-action@v4
```
Comment (JcMinarro) [inline .github/workflows/ci.yml:20]: Why is it commented on?

File: build.gradle [hunk 2]
```diff
@@ -6,7 +6,7 @@ plugins {
 }
 
 group 'io.getstream.client'
-version = '3.6.2'
+version = '3.7.0'
```
Comment (JcMinarro) [inline build.gradle:9]: Version shouldn't be manual update.
There is a CI job for that

**Review by JcMinarro — CHANGES_REQUESTED** [review]
The commit message history doesn't follow the expected structure.
On this repository we are using [CommitLint](https://commitlint.js.org/) to check the commit message.

**Comment by JcMinarro** [comment]
> Hi @xernobyl @JcMinarro when will this changes can be release in new version? Thanks.

We need this PR pass the CI checks, could you fix them?

**Comment by adinovri** [comment]
Hi @JcMinarro @xernobyl please review here https://github.com/GetStream/stream-java/pull/127, I will close this PR. Thanks.

**Pull request closed without merging**